Sources/DistributedActorsTestKit/TestProbes.swift

```diff
@@ -767,3 +776,104 @@ extension ActorTestProbe {
         self.internalRef.tell(.stopCommand)
     }
 }
+
+// ==== ----------------------------------------------------------------------------------------------------------------
+// MARK: Distributed actor lifecycle watch
+
+extension ActorTestProbe {
+    /// Instructs this probe to watch the passed in actor.
+    /// The `watchee` actor is from now on being watched and we will be notified when it terminates.
+    ///
+    /// There is no difference between keeping the passed in reference or using the returned ref from this method.
+    /// The actor is the being watched subject, not a specific reference to it.
+    ///
+    /// This enables it to use `expectTerminated` to await for the watched actors termination.
+    ///
+    /// - Returns: reference to the passed in `watchee` actor.
+    /// - SeeAlso: `DeathWatch`
+    @discardableResult
+    public func watch<Act>(
+        _ watchee: Act,
+        file: String = #filePath, line: UInt = #line
+    ) async -> Act
+        where Act: DistributedActor,
+        Act.ActorSystem == ClusterSystem
+    {
+        _ = await self._internal.whenLocal { __secretlyKnownToBeLocal in // TODO(distributed): this is annoying, we must track "known to be local" in typesystem instead
+            __secretlyKnownToBeLocal.watchTermination(of: watchee, file: file, line: line)
+        }
+        return watchee
+    }
+
+    /// Checks that the actor has been terminated.
+    ///
+    /// - Warning: Remember to first `watch` the actor you are expecting termination for,
+    ///            otherwise the termination signal will never be received.
+    ///
+    /// - SeeAlso: `DeathWatch`
+    // TODO: expectTermination(of: ...) maybe nicer wording?
```

Comment:
either works for me, whichever you prefer :)

Reply:
Maybe you're right actually -- Terminated wording is from "old world" where it literarily was "the Terminated" message